### PR TITLE
Add vsphere section for openshift_node_kubelet_args_dict

### PR DESCRIPTION
**Pull request by ganhuang**
Fix https://bugzilla.redhat.com/show_bug.cgi?id=1543830

### CODE REVIEW

**Comment by sdodson** [comment]
@dav1x can you take a look at this?

**Comment by dav1x** [comment]
@ganhuang tested and works great! Thanks

@sdodson can you merge this please? I've got some additions on top of this to address the 3.9 service account issues.

File: roles/openshift_node/defaults/main.yml [hunk 1]
```diff
@@ -54,6 +54,12 @@ openshift_node_kubelet_args_dict:
     cloud-config:
     - "{{ openshift_config_base ~ '/cloudprovider/azure.conf' }}"
     node-labels: "{{ l_node_kubelet_node_labels }}"
+  vsphere:
+    cloud-provider:
+    - vsphere
```
Comment (sdodson) [inline roles/openshift_node/defaults/main.yml]: The tests are failing due to the trailing whitespace on this line

**Comment by ganhuang** [comment]
thanks, I think now it's fixed.

**Pull request merged**